Makefile

```diff
@@ -138,7 +138,7 @@ ifeq (, $(shell which golangci-lint))
 	exit 1
 endif
 
-	golangci-lint run --timeout 5m0s --issues-exit-code 0
```

Comment:
I'm concerned that enabling many linters will increase time too much.  Does removing this mean we expect it to take longer than 5 minutes?  Instead of removing the timeout maybe we should increase it.

Reply:
@reimda this setting was just moved to `.golangci.yml`

Reply:
Aha, thanks